Dfe.Data.SearchPrototype/Web/Tests/Shared/SearchServiceAdapter/DummySearchServiceAdapter.cs

```diff
@@ -22,7 +22,7 @@ public async Task<EstablishmentResults> SearchAsync(SearchContext searchContext)
 
             IEnumerable<Establishment> establishments =
                 from establishmentToken in establishmentsObject["establishments"]
-                where establishmentToken["name"]!.ToString().Contains(searchContext.SearchKeyword)
+                where (establishmentToken["name"]!.ToString()+"*").Contains(searchContext.SearchKeyword)
```

Comment:
Not quite sure what this part does here in the test dummy implementation.

Does something break if the `+"*"` is not included?